### ISSUE TITLE
How do I configure the extension?

### ISSUE THREAD

**Issue by marknotton** [Teamwork/vscode-projects#27]
This may be a really simple thing for most people to grasp, so apologies for asking what might be an obvious request... but there aren't any guidelines on how to actually implement the extension configurations. 

The project overview suggests we must configure the Extension Settings first. Can you explain -- in layman's terms -- how to do this, please? 

Standard Google searches haven't offered any clear solutions.

**Comment by Flexicon** [comment]
Hi @marknotton , sorry for the late response. In "layman's terms" the "configuration" process is just authenticating with Teamwork and selecting a project for your workspace. The wording in the readme could use some work. 😅

**Comment by marknotton** [comment]
Ha, I knew I was overthinking this! Thanks for clearing that up for me 👍
